src/storyblok/Project.astro

```diff
@@ -0,0 +1,147 @@
+---
+import { renderRichText } from '@storyblok/astro';
+import { Image } from 'astro:assets';
+import StoryblokComponent from '@storyblok/astro/StoryblokComponent.astro'
+
+const { blok, name, projects } = Astro.props;
+
+import experiments from '../../src/images/svgs/experiments.svg';
+import stunts from '../../src/images/svgs/stunts.svg';
+import tools from '../../src/images/svgs/tools.svg';
+
+const fallback = {
+  experiments,
+  stunts,
+  tools
+}[blok.type.toLowerCase()] || experiments;
+
+function renderContent(content) {
+  return content ? renderRichText(content) : '';
+}
+---
+
+<article class="project">
+
+  <!-- Project Title -->
+  <h1>{name}</h1>
+
+  <!-- Project Logo or Fallback Image -->
+  {blok.logo?.filename ? (
+    <Image
+      src={blok.logo.filename}
+      inferSize={true}
+      alt=""
+    />
+  ) : (
+    <Image
+      src={fallback}
+      inferSize={true}
+      alt=""
+    />
+  )}
+
+  <!-- Project Details -->
+  <ul>
+    {blok.url && (
+      <li>
+        <a href={blok.url}>
+          {blok.url.replace(/^(https?:\/\/)?(www\.)?/i, '').split('/')[0]}
+        </a>
+      </li>
+    )}
+    {blok.xHandle && (
+      <li>
+        <a href={`https://x.com/${blok.xHandle}`} rel="noreferrer">@{blok.xHandle}</a>
+      </li>
+    )}
+    {blok.type && (
+      <li>
+        {blok.type}
+      </li>
+    )}
+    {blok.launchDate && (
+      <li>
+        {new Date(blok.launchDate).toLocaleDateString('en-US', { month: '2-digit', year: 'numeric' }).replace('/', '/')}
+      </li>
+    )}
+    <li>
+      {blok.retiredDate ? 'archived' : 'active'}
+    </li>
+  </ul>
+
+  <!-- Project Description -->
+  {blok.description && <section set:html={renderContent(blok.description)} />}
+
+  <!-- Example Blocks -->
+  {blok.examples && blok.examples.length > 0 && (
+    <>
```

Comment:
I'm not sure if you actually need these Fragments.

Reply:
I think I do since they're wrapping two things? Though this is just roughing in data integration and I'll hit this with more 🧠 when I come back for markup and styling.

Reply:
That's just a React thing. Shouldn't have to use them in Astro.